web/Web/web/static/js/document-viewer.js

```diff
@@ -52,12 +53,15 @@ var docView = function() {
     documentJudgingCriteriaButtonGroupSelector: ".judging-criteria-btn-group",
     documentHRelButtonSelector: ".docViewDocHRelButton",
     documentRelButtonSelector: ".docViewDocRelButton",
+    docViewNextDocButtonSelector: ".docViewNextDocButton",
+    docViewPreviousDocButtonSelector: ".docViewPreviousDocButton",
     documentNonRelButtonSelector: ".docViewDocNonRelButton",
     documentAdditionalJudgingCriterionSelector: ".additionalJudgingCriterion",
     previouslyReviewedListSelector: ".previouslyReviewedList",
     previouslyReviewedListSpinnerSelector: ".previouslyReviewedListSpinner",
     searchItemSelector: ".searchItemSelector",
     documentModalSelector: "#documentModal",
+    sortReviewedDocumentsSelector: ".sort_by_relevance",
```

Comment:
Minor edit. Let's rename this to `docViewSortReviewListByRelButton` just to be consistent with the naming.

Reply:
fixed